node/sc/metrics.go

```diff
@@ -43,10 +43,11 @@ var (
 	vtHandleEventMeter  = metrics.NewRegisteredMeter("klay/bridge/vt/event/handle", nil)
 
 	vtRecoveredRequestEventMeter = metrics.NewRegisteredMeter("klay/bridge/vt/event/recovery/request", nil)
-	vtPendingRequestEventMeter   = metrics.NewRegisteredCounter("klay/bridge/vt/event/pend/request", nil)
+	vtPendingRequestEventCounter = metrics.NewRegisteredCounter("klay/bridge/vt/event/pend/request", nil)
```

Comment:
```suggestion
	vtPendingRequestEventCounter = metrics.NewRegisteredCounter("klay/bridge/vt/event/pending/request", nil)
```

Reply:
After this change, grafana json file should be modified. So I don't want to modify this at this time.